core/trino-server-rpm/pom.xml

```diff
@@ -116,7 +116,6 @@
             <plugin>
                 <groupId>org.codehaus.gmaven</groupId>
                 <artifactId>groovy-maven-plugin</artifactId>
-                <version>2.1.1</version>
```

Comment:
separate commit?

Reply:
This just moves the version to the root POM.